launch/hsr_demo.launch

```diff
@@ -0,0 +1,24 @@
+<!--
+  Copyright 2016 Toyota Research Institute
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<launch>
+  <param name="/use_sim_time" value="true" />
+  <include file="$(find cartographer_hsr)/launch/hsr.launch" />
+  <node name="rviz" pkg="rviz" type="rviz" required="true"
```

Comment:
I always wonder how polluted we should be making the package.xml with dependencies.  This example launch adds "rviz", but is not necessary to actually use the package (unless you want to run the demo).